coa-helm/templates/cromwell.yaml

```diff
@@ -36,16 +38,18 @@ status: {}
 {{ end }}
 
 ---
-
-{{ include "terra-batch-libchart.cromwell-service" (list . "coa-helm.cromwell-service") }}
+{{- if .Values.cromwell.coaEnabled }}
+{{ include "terra-batch-libchart.cromwell-config" (list . "cromwell-helm.cromwell-config") }}
+{{ end }}
```

Comment:
this changed `terra-batch-libchart.cromwell-service` to `terra-batch-libchart.cromwell-config` - a cut-and-paste error maybe?

Reply:
great catch here -- updated!